airflow/ti_deps/deps/mapped_task_expanded.py

```diff
@@ -26,6 +26,8 @@ class MappedTaskIsExpanded(BaseTIDep):
     IS_TASK_DEP = False
 
     def _get_dep_statuses(self, ti, session, dep_context):
```

Comment:
is there / should there be any  interaction with  the IGNORABLE attribute here?

Reply:
No - ignorable is mostly for use when using `airflow task run --ignore-all` to make a task run anyway, even if a dep (depends on last, or states, say) is saying it shouldn't run.

But this dep can't be skipped - because the task _will never run_ until it's been expanded.